stdlib/Dates/docs/src/index.md

```diff
@@ -710,6 +710,29 @@ Dates.dayofweek
 Dates.dayofmonth
 Dates.dayofweekofmonth
 Dates.daysofweekinmonth
+Dates.January
+Dates.Jan
+Dates.February
+Dates.Feb
+Dates.March
+Dates.Mar
+Dates.April
+Dates.Apr
+Dates.May
+Dates.June
+Dates.Jun
+Dates.July
+Dates.Jul
+Dates.August
+Dates.Aug
+Dates.September
+Dates.Sep
+Dates.October
+Dates.Oct
+Dates.November
+Dates.Nov
+Dates.December
+Dates.Dec
```

Comment:
These are all documented in a nice table below. I suggest leaving out these ones from the manual, and just keep the table.

Reply:
Onto it!

Reply:
So, I delete this entirely, right?

Reply:
Yea.

Reply:
Done!